.vitepress/config.ts

```diff
@@ -679,9 +684,19 @@ export default defineConfigWithTheme<ThemeConfig>({
         text: 'Português',
         repo: 'https://github.com/vuejs-translations/docs-pt'
       },
+      {
+        link: 'https://bn.vuejs.org',
+        text: 'বাংলা',
+        repo: 'https://github.com/vuejs-translations/docs-bn'
+      },
+      {
+        link: 'https://it.vuejs.org',
+        text: 'Italiano',
+        repo: 'https://github.com/vuejs-translations/docs-it'
+      },
       {
         link: '/translations/',
-        text: 'Help Us Translate!',
+        text: 'Aiutaci a Tradurre!',
```

Comment:
Pura e inutile opinione personale: il text-capitalize in italiano è proprio brutto, se possibile eviterei